cmd/libs/go2idl/make-client/output/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
According to the client proposal, in the future there will be files for each resource in that group/version. Will that mitigate the uglyness?